doc/content/reference/configuration/the-things-stack.md

```diff
@@ -49,6 +49,10 @@ When running a cluster in a trusted network, you can allow sending credentials o
 
 - `grpc.allow-insecure-for-credentials`: Allow transmission of credentials over insecure transport
 
+You can suppress log messages for successful gRPC method calls (e.g. to reduce the noise caused by the health checks in a production environment).
+
+- `grpc.log-ignore-methods`: List of gRPC methods for which to suppress logs of successful requests.
```

Comment:
Maybe good to clarify that if they see too many `grpc_method=MethodName grpc_service=ttn.lorawan.v3.ServiceName` in the logs, they should configure `/ttn.lorawan.v3.ServiceName/MethodName` here (I think).